spec/association_spec.rb

```diff
@@ -7,8 +7,6 @@
     remove_all_models
   end
 
-  let(:pk_type) { Gem::Requirement.new('< 5.1').satisfied_by?(::ActiveRecord.version) ? :integer : :bigint }
```

Comment:
there are other 2 branches below in this file that you might want to remove 

[ActiveRecord::VERSION](https://github.com/SchemaPlus/schema_associations/blob/709f4c7134951ec793b51e0d3ef1c612275a72b5/spec/association_spec.rb#L384

https://github.com/SchemaPlus/schema_associations/blob/709f4c7134951ec793b51e0d3ef1c612275a72b5/spec/association_spec.rb#L360)

Reply:
Thanks!.. I'll remove those